apps/web/src/server/views/applications/case/overview-legacy.njk

```diff
@@ -55,9 +55,9 @@
       }
```

Comment:
we should have the `Material change` field displayed based on ht eFF status in the overview-legacy as well

Reply:
@francescobrenci-AR I might be mistaken as I thought when the 'applic-55-welsh-translation' flag is false but 'applics-156-material-changes' is true the 'is material change' field appears on the 'project information' page. If 'applic-55-welsh-translation' is true it then appears on the new overview page. Is it your understanding that this is not the case?

Reply:
yes - that's correct, that is the expected behaviour